server/service/testing_utils.go

```diff
@@ -224,3 +228,21 @@ func testStdoutPluginConfig() config.FleetConfig {
 	c.Osquery.StatusLogPlugin = "stdout"
 	return c
 }
+
+func assertBodyContains(t *testing.T, resp *http.Response, expectedError string) {
```

Comment:
Seems like this could just as well be used to assert a successful call, the expected string could just be called `expected`?